test/EntityFramework/FunctionalTests.netcoreapp3.0/ProductivityApi/SimpleScenariosForLocalDb.cs

```diff
@@ -0,0 +1,427 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved. See License.txt in the project root for license information.
+
+#if !NET40
+
+namespace ProductivityApiTests
+{
+    using System;
+    using System.Data.Common;
+    using System.Data.Entity;
+    using System.Data.Entity.Infrastructure;
+    using System.Data.SqlClient;
+    using System.IO;
+    using System.Linq;
+    using System.Transactions;
+    using SimpleModel;
+    using Xunit;
+
+    public class SimpleScenariosForLocalDb : FunctionalTestBase, IDisposable
+    {
+        #region Infrastructure/setup
+
+        private readonly object _previousDataDirectory;
+
+        public SimpleScenariosForLocalDb()
+        {
+            // TODO: Can we hardcode this somewhere like we do for the EF provider?
+            DbProviderFactories.RegisterFactory("System.Data.SqlClient", SqlClientFactory.Instance);
```

Comment:
All the test files were just coppied and trimmed to make them compile.

This is the only line added. Users will need to put this in their app startup for Preview 5.

Reply:
I guess we could add fallback logic every time we ask for the factory.

Reply:
I think we already have the indirection; I just didn’t want to rush the change. I’ll investigate Monday morning to see if there’s a simple change we can make to Preview 5.